### PR TITLE
Aligned and size decreased.

### LINKED ISSUE #1044: 🐞[Bug]: proper aligning
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Patient Portal 

Size decrement of the container

### Feedback Section

size decrement if your feedback input

### Add ScreenShots

Below ss from the deploy of the repo.

image

image

### What browsers are you seeing the problem on?

Chrome

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#1044

## Description

[Please include a brief description of the changes or features added]

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #1044 

## Type of PR

- [X] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)

https://github.com/user-attachments/assets/404234fa-9f75-4e5d-97c3-886f00293628

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR , preview video added in PR description is deployment view.

File: js/emailvalidation.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+emailjs.init("Your_User_ID"); // Replace with your actual EmailJS User ID
```
Comment (varshith257) [inline js/emailvalidation.js:1]: Export the function and use it where it used before this fn

Comment (Anjaliavv51) [reply js/emailvalidation.js:1]: I didn't get u.

Comment (Anjaliavv51) [reply js/emailvalidation.js:1]: @varshith257 could u explain clearly.

File: js/emailvalidation.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    
+        // Function to validate email address
+        function isValidEmail(email) {
```
Comment (varshith257) [inline js/emailvalidation.js:4]:
```suggestion
       export function isValidEmail(email) {
```

**Review by varshith257 — COMMENTED** [review]
So that you can use it in index.html to validate email and.other files too

**Comment by Anjaliavv51** [comment]
@varshith257 once review it again

**Pull request merged**